src/cli/self_update.rs

```diff
@@ -75,9 +75,10 @@ This will download and install the official compiler for the Rust
 programming language, and its package manager, Cargo.
 
 It will add the `cargo`, `rustc`, `rustup` and other commands to
-Cargo's bin directory, located at:
+Cargo's bin directory, under the Cargo home directory, which is
+located at:
 
-    {cargo_home_bin}
+    {cargo_home}
 
 This can be modified with the CARGO_HOME environment variable.
```

Comment:
```suggestion
This can be modified with the CARGO_HOME environment variable
The cargo's bin directory is always under $CARGO_HOME/bin.
```

Reply:
OK, merged.

Reply:
I don't quite happy with my suggestion. 
The old `cargo_home_bin` is correct. But the paragraph needs to be rephrased.

Reply:
_How about below one?_

This will download and install the official compiler for the Rust
programming language, and its package manager, Cargo.

The Cargo's home directory located at:

    {cargo_home}

This can be modified with the CARGO_HOME environment variable.

The `cargo`, `rustc`, `rustup` and other commands will be added to
Cargo's bin directory, located at:

    {cargo_home_bin}

Reply:
The last commit with below message shown:

<img width="499" alt="屏幕快照 2020-04-25 下午6 35 32" src="https://user-images.githubusercontent.com/159167/80277780-045d0980-8724-11ea-8847-94c7335a9073.png">